recipe/meta.yaml

```diff
@@ -11,8 +11,8 @@ source:
   sha256: 25462a009dd7fa9a48520c56a691813a62919a6c842784d6ed564fe7f9f9bcf8
 
 build:
-  number: 0
-  noarch: python
+  number: 1
+  noarch: python =2.7|>=3.4
```

Comment:
Sorry if I wasn't clear. The version restriction just goes under the `requirements` section for both `host` and `build`, not the `noarch`

Reply:
Sorry, my bad, you were clear, I accidentally modified the wrong line.